packages/react/src/pages/crm-contact-list/contact-panel/contactPanel.tsx

```diff
@@ -0,0 +1,225 @@
+                      value={contact?.manager}
+                      isEditing={!isEditing}
+                      onValueChange={updateField('manager')}
+                    />
+                  </FormItem>
+                </FormItem>
+              </FormItem>
+
+              <FormItem itemType='group' cssClass='contact-fields-group'>
+                <FormItem>
+                  <FormTextbox
+                    value={contact?.phone}
+                    isEditing={!isEditing}
+                    onValueChange={updateField('phone')}
+                    icon='tel'
+                    mask='+1(000)000-0000'
+                  />
+                </FormItem>
+                <FormItem>
+                  <FormTextbox
+                    value={contact?.email}
+                    isEditing={!isEditing}
+                    onValueChange={updateField('email')}
+                    icon='email'
+                  />
+                </FormItem>
+                <FormItem>
+                  <FormTextbox
+                    value={contact?.address}
+                    isEditing={!isEditing}
+                    onValueChange={updateField('address')}
+                    icon='home'
+                  />
+                </FormItem>
+              </FormItem>
+            </Form>
+          </div>
+
+          <div className='data-part data-part-toolbar border'>
+            <Toolbar>
+              <ToolbarItem location='before' visible={!isEditing}>
+                <Button icon='edit' text='Edit' stylingMode='outlined' type='default' onClick={toggleEdit} />
+              </ToolbarItem>
+              <ToolbarItem location='before' visible={!isEditing}>
+                <Button text='Details' stylingMode='outlined' type='default' onClick={navigateToDetails} />
+              </ToolbarItem>
+              <ToolbarItem location='before' locateInMenu='before' visible={isEditing}>
+                <Button text='Save' stylingMode='outlined' type='default' onClick={toggleEdit} />
+              </ToolbarItem>
+              <ToolbarItem location='before' locateInMenu='before' visible={isEditing}>
+                <Button text='Cancel' stylingMode='text' onClick={toggleEdit} />
+              </ToolbarItem>
+              <ToolbarItem location='after' visible={!isEditing}>
+                <DropDownButton text='Actions' width={120} stylingMode='contained' items={['Call', 'Send Fax', 'Send Email', 'Make a Meeting']} />
+              </ToolbarItem>
+            </Toolbar>
+          </div>
+          <div className='data-part'>
+            <Accordion multiple collapsible itemTitleRender={customTitle}>
+              <AccordionItem title='Opportunities' render={customOpportunities} />
+              <AccordionItem title='Activities' render={customActivities} />
+            </Accordion>
+          </div>
+        </ScrollView>
+      </div>
+      { loading && <LoadPanel container='.panel' visible position={{ of: '.panel' }} /> }
```

Comment:
Wrap component in our custom LoadPanel. This reduce the code and prevent of using contact?... Also this should fix undefined text for ContactStatus component